src/Microsoft.Azure.WebJobs.Extensions.Dapr/Services/DaprServiceListener.cs

```diff
@@ -78,6 +93,11 @@ public async Task EnsureStartedAsync(CancellationToken cancellationToken)
                 this.logger.LogInformation($"Starting Dapr HTTP listener on {this.appAddress} with {this.listeners.Count} function listener(s) registered.");
                 await this.host.StartAsync(cancellationToken);
                 this.logger.LogInformation("Dapr HTTP host started successfully.");
+
+                if (this.enableSidecarCheckOnHostStartup)
```

Comment:
Can we please add unit test for this feature flag?

Reply:
Are you proposing to test `IsSidecarMetadataCheckOnHostStartupDisabled`?

Reply:
Yeah, having the test should ensure if somebody changes default value from false to true, the test should fail. One for default value test and one for setting it to true.

Reply:
Added test for the flag.